services/src/handlers/datasets.rs

```diff
@@ -331,7 +339,7 @@ fn detect_time_type(layer: &Layer, columns: &Columns) -> OgrSourceDatasetTimeTyp
 
     let mut start = None;
     let mut end = None;
-    for column in &(columns.textual) {
+    for column in columns.textual.iter().chain(&(columns.date)) {
```

Comment:
This is strange since if you have a date column, you know that this is a date.
But here, it can fail (and it does?).

I guess you should at least see if `into_string` transforms dates into rfc3339 dates. Otherwise, it will always fail.

Reply:
only because Gdal thinks the column is a datetime it doesn't mean that it is an ISO time. And I only suggest columns containing ISO datetime as a start or end column because I don't know the format String I would have to provide to the user in the OgrSourceTimeFormat

Reply:
only because Gdal thinks the column is a datetime it doesn't mean that it is an ISO time. And I only suggest columns containing ISO datetime as a start or end column because I don't know the format String I would have to provide to the user in the OgrSourceTimeFormat

Reply:
Are dates then really parsed by that format string or is it just the date used then? In the latter case, we could do it anyway.

Reply:
we always parse dates ourselves and don't use the Gdal parsing. https://github.com/geo-engine/geoengine/blob/master/operators/src/source/ogr_source.rs#L491